.github/workflows/gotests.yml

```diff
@@ -88,3 +88,11 @@ jobs:
         name: tetragon-bugtool
         path: /tmp/tetragon-bugtool*
         retention-days: 5
+
+    - name: Upload vmlinux file
+      if: failure()
+      uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4.6.0
+      with:
+        name: vmlinux-${{ matrix.os }}
+        path: /sys/kernel/btf/vmlinux
+        retention-days: 5
```

Comment:
These are just the BTF definitions for the kernel, not the kernel itself, right?

Note that you can get the btf files from the lvh images.
For example:
```
x=$(docker create --platform linux/arm64  quay.io/lvh-images/kernel-images:6.12-20250319.212144)
docker cp $x:/data/kernels /tmp/deleteme
ls /tmp/deleteme/6.12/boot/btf-6.12.19
```

Reply:
Yes it's just the 1.5MB BTF file

Thanks for this workaround solution ! I'll have a look

Reply:
Then, let's name the file so that it's clear that it's the btf file? (e.g., btf-vmlinux-${{ matrix.os }})?